src/v0/destinations/am/transform.js

```diff
@@ -528,14 +528,31 @@ function processSingleMessage(message, destination) {
 
   const messageType = message.type.toLowerCase();
   const CATEGORY_KEY = 'properties.category';
+  const { useUserDefinedPageEventName, userProvidedPageEventString } = destination.Config;
   switch (messageType) {
     case EventType.IDENTIFY:
       payloadObjectName = 'events'; // identify same as events
       evType = 'identify';
       category = ConfigCategory.IDENTIFY;
       break;
     case EventType.PAGE:
-      evType = `Viewed ${message.name || get(message, CATEGORY_KEY) || ''} Page`;
+      if (useUserDefinedPageEventName) {
```

Comment:
@aashishmalik when does migration to add `useUserDefinedPageEventName` field will be done ?

Reply:
Thursday, as this is a new boolean with defualt as false.